src/Uno.Extensions.Validation/IValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Uno.Extensions.Validation;
+
+/// <summary>
+/// Defines an interface for a data validator.
+/// </summary>
+/// <typeparam name="T">Instance to validate</typeparam>
+public interface IValidator<in T>
```

Comment:
Please see my comment on the discussion https://github.com/unoplatform/uno.extensions/discussions/324#discussioncomment-4380008 :)